builders/interoperability/xcm/fees.md

```diff
@@ -60,18 +60,28 @@ Substrate已推出一个权重系统，决定一个函数的权重，也就是
 
 ### Polkadot {: #polkadot }
 
-如同先前提到的，波卡目前对所有XCM指令采取[固定权重数量](https://github.com/paritytech/polkadot/blob/{{networks.polkadot.spec_version}}/runtime/polkadot/src/xcm_config.rs#L111){target=_blank}的计算方式，也就是每条指令`{{ networks.polkadot.xcm_instructions.weight.display }}`权重单位。
+The total weight costs on Polkadot take into consideration database reads and writes in addition to the weight required for a given instruction. Polkadot uses benchmarked weights for instructions and database read and write operations. The breakdown of weight costs for the database operations is as follows:
```

Comment:
Assume this will be translated in the future

Reply:
oops forgot about this!

@wuzhong-papermoon can you please translate this?